src/System.Device.Ports/System.Device.Ports.SerialPort/ISR.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.Ports.SerialPort.Resources;
+using System.Resources;
+
+namespace System.Device.Ports.SerialPort
+{
+    /// <summary>
+    /// Error Messages
+    /// </summary>
+    internal class ISR
```

Comment:
We shouldn't need to create our own handling of SR for the resources. We should instead just use the traditional resource manager APIs from the code as any normal library.